spec.html

```diff
@@ -119,9 +119,9 @@ <h1>AggregateError ( _errors_, _message_ )</h1>
         <p>When the *AggregateError* function is called with arguments _errors_ and _message_, the following steps are taken:</p>
         <emu-alg>
           1. If NewTarget is *undefined*, let _newTarget_ be the active function object, else let _newTarget_ be NewTarget.
-          1. Let _O_ be ? OrdinaryCreateFromConstructor(_newTarget_, `"%AggregateError.prototype%"`, « [[ErrorData]], [[AggregateErrors]] »).
+          1. Let _O_ be ? OrdinaryCreateFromConstructor(_newTarget_, `"%AggregateError.prototype%"`, « [[ErrorData]] »).
           1. Let _errorsList_ be ? IterableToList(_errors_).
-          1. Set _O_.[[AggregateErrors]] to _errorsList_.
+          1. Perform ! CreateDataPropertyOrThrow(_O_, `"errors"`, _errorsList_).
```

Comment:
I think a call to CreateArrayFromList is missing here.

Reply:
https://www.youtube.com/watch?v=hou0lU8WMgo